src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto

```diff
@@ -90,3 +94,30 @@ message RefuseRequisitionRequest {
   // Details about the refusal. Required.
   Requisition.Refusal refusal = 2;
 }
+
+// Request message for `FulfillRequisition` method.
+message FulfillRequisitionRequest {
+  // Resource name of the `Requisition. Required.
+  string name = 1
+      [(google.api.resource_reference).type = "halo.wfanet.org/Requisition"];
+
+  // Encrypted `SignedData` containing the data for fulfillment. Required.
```

Comment:
I realized that there should be additional info here in the comment to clarify. Something like
  //
  // The encryption uses the `measurement_public_key` from `measurement_spec` as
  // the recipient public key.